sdks/python/setup.py

```diff
@@ -85,7 +85,6 @@ def get_version():
         'apache_beam/utils/counters.py',
         'apache_beam/utils/windowed_value.py',
     ]),
-    setup_requires=['nose>=1.0'],
```

Comment:
@aaltay the problem is that `setup_requires` gets the dependency in `./.eggs/`, which was causing problems with RAT about unlicensed files. That's why I move to manage it with `tox`.

If you know any way to customize that path (e.g., `./target/eggs/`) then I'm happy to revert it.